README.md

```diff
@@ -188,3 +188,24 @@ Then, just add breakpoints by clicking on the line number of your code.
 9. If you'd like to share the query you built, such as demonstrating how you tested your code, check out [Apollo Explorer's sharing features](https://www.apollographql.com/blog/announcement/platform/save-and-share-your-graphql-operations-in-apollo-explorer/#sharing-a-collection).
 
 </details>
+
+<details>
```

Comment:
I'm a little confused about why these steps are needed. Generally, tests are something that we should be able to just run and not have to manually do anything in between.

Reply:
@theyokohamalife This was something added to the other branch that came along when I created this new branch (still compared to `main`). Firebase requires these indexes for every possible combination of `fieldToOrder` and `orderDirection`. There is a way to automate their creation, but it looks a bit difficult to do. But I'm going to have a crack at it this morning and hopefully we won't need this section.